docker/k8s-first-steps/run_server.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
-echo "out/logs is going into '/app/run_webApp.log'"
+echo "out/logs is going into '/app/run_server.log'"
```

Comment:
Not sure about the log file, but I guess we should rename this file to `server_run.sh` to be consistent with the naming in the parent folder.

Reply:
Renamed.